addr2line.c

```diff
@@ -2634,7 +2634,11 @@ rb_dump_backtrace_with_lines(int num_traces, void **traces, FILE *errout)
             memcpy(main_path, binary_filename, len+1);
             append_obj(&obj);
             obj->path = main_path;
+#ifdef USE_ELF
+            addr = fill_lines(num_traces, traces, 1, &obj, lines, -1, errout);
```

Comment:
Wouldn't it make more sense to make the `fill_lines` function defined for Mach-O handle the `-1` offset (and convert it to `0`) too, instead of knowing how to exactly call the function depending on the platform?

Reply:
Yeah, that makes sense to me. I'll update it.